README.md

```diff
@@ -73,96 +68,10 @@ My first project at Microverse
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
 - [Live Demo Link](https://google.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
```

Comment:
- [ ] In the **_Prerequisite section_** of your README file, kindly add packages or applications that would be needed to start this application on the local machine. You can include NPM, a browser, a code editor, git and any other thing you can think of. 👍